scripts/software-versions-inventory/README.md

```diff
@@ -0,0 +1,55 @@
+# Script: Software Versions Inventory
+
+## Prerequisites
+
+### `cargo`
+
+Install `cargo` by installing the Rust toolchain via [rustup](https://rustup.rs).
+
+### `make`
+
+If you do not have it installed, install `make` on your system.
+On fedora, run `sudo dnf install make`.
+
+### `rpmspec`
+
+On fedora, run `sudo dnf install rpm-build` to install the `rpmspec` command, which is used in the software versions inventory script.
+
+### `bottlerocket-license-tool`
+
+1. Clone the [Bottlerocket SDK repository](https://github.com/bottlerocket-os/bottlerocket-sdk)
+2. `cd` into the directory where you cloned the SDK
+3. Install the Bottlerocket license tool with `cargo install bottlerocket-license-tool --path ./license-tool/`
+
+### Fetching licenses
+
+1. Place a file named `Licenses.toml` in the root of your Bottlerocket OS repository with the following contents:
+
+```toml
```

Comment:
Is there a way to not have to put in these faux license files? E.g. handle the error produced by the nvidia pacakging?

Reply:
The solution seems to be skipping packages that call out to `bottlerocket-license-tool`. Adding a note about that